api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/ClaimServiceTest.java

```diff
@@ -36,19 +38,25 @@
     @Mock
     private EligibilityStatusCalculator eligibilityStatusCalculator;
 
-    @Test
-    public void shouldSaveNewClaimant() {
+    @ParameterizedTest(name = "Should save claimant with claim status set to {1} when eligibility status is {0}")
+    @CsvSource({
+            "ELIGIBLE, NEW",
```

Comment:
we need INELIGIBLE here too. Can we add a test that uses an EligibilityStatus EnumSource and confirms that the persisted claimStatus is not null?

Reply:
Added missing case.